.github/workflows/release-packages.yaml

```diff
@@ -221,10 +221,13 @@ jobs:
         id: get_release_info
         uses: bruceadams/get-release@v1.2.0
       - name: Set Image Tag
-        run: echo "IMAGE_TAG=diffblue/cbmc:${{ steps.get_release_info.outputs.tag_name }}" >> $GITHUB_ENV
+        run: |
+          VERSION = ${{ steps.get_release_info.outputs.tag_name }}
+          arrVERSION=(${VERSION//-/ })
+          echo "IMAGE_TAG=diffblue/cbmc:${arrVERSION[1]}" >> $GITHUB_ENV
       - name: Build docker image
         run: docker build -t "$IMAGE_TAG" .
       - name: Push docker image to DockerHub
         run: |
           docker login --username=dbcicprover --password=${{ secrets.DOCKERHUB_ACCESS_DB_CI_CPROVER }}
```

Comment:
Would there be any overheads of keeping username in github secrets too? I think it might be easier to maintain if we ever change the username if it is (or is going to be) used in multiple places. It would make the credentials more secure too.

Reply:
I asked Peter about it, but he said that this doesn't need to be a secret.

Reply:
OK, do you think you will need this in any other place so that it would be worth of keeping it as a define somewhere above?

Reply:
I don't think it's going to be needed anywhere else to be honest.

And if it is, I can do the refactoring later when there is evidence it's needed.

Reply:
The benefit of keeping the username as a secret would be that forks of CBMC could supply different secrets rather than diverge the code. However it shouldn't make any difference to us.